ports/unix/mpconfigport.h

```diff
@@ -98,6 +98,7 @@
 #define MICROPY_PY_SYS_MAXSIZE      (1)
 #define MICROPY_PY_SYS_STDFILES     (1)
 #define MICROPY_PY_SYS_EXC_INFO     (1)
+#define MICROPY_PY_SYS_BUILTIN_MODULE_NAMES (1)
```

Comment:
Should be 0, i.e. shouldn't be there.

Reply:
hmm, okay. will remove.

Reply:
Instead should be in mpconfigport_coverage.h for tests. There's also https://github.com/micropython/micropython/pull/4309 for "let's enable everything and more!".

Reply:
mpconfigport_coverage.h is what's used in the CI later on for `tests/run-tests`?

Reply:
mpconfigport_coverage.h's primary purpose is to produce "coverage: 98%" figure you can see at https://github.com/micropython/micropython (in README). Yes, it does that by running all the usual tests, plus some artificial, to exercise code paths which can't be reached otherwise. So in all, mpconfigport_coverage.h must enable as much as possible (ideally, all) features implemented.

Reply:
got it, will update